plugins/destination/sqlite/client/client_test.go

```diff
@@ -1,32 +1,32 @@
 package client
 
 import (
+	"context"
+	"encoding/json"
 	"testing"
 
-	"github.com/cloudquery/plugin-pb-go/specs"
-	"github.com/cloudquery/plugin-sdk/v3/plugins/destination"
+	"github.com/cloudquery/plugin-sdk/v4/plugin"
 )
 
-var migrateStrategy = destination.MigrateStrategy{
-	AddColumn:           specs.MigrateModeSafe,
-	AddColumnNotNull:    specs.MigrateModeForced,
-	RemoveColumn:        specs.MigrateModeSafe,
-	RemoveColumnNotNull: specs.MigrateModeForced,
-	ChangeColumn:        specs.MigrateModeForced,
-}
-
 func TestPlugin(t *testing.T) {
-	destination.PluginTestSuiteRunner(t,
-		func() *destination.Plugin {
-			return destination.NewPlugin("sqlite", "development", New)
-		},
-		specs.Destination{
-			Spec: &Spec{
-				ConnectionString: ":memory:",
+	ctx := context.Background()
```

Comment:
```suggestion
	ctx := context.TODO()
```
as `context.TODO()` exists for this exact reason

Reply:
not sure I understand what is the reasoning here. moreover in a golang `Test` doens't provide a context so it's not a TODO it's background (which is the same thing just that TODO indicates we want to change in the future while we don't)